app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,16 @@
+class AccountActivationsController < ApplicationController
+
+  def edit 
+    user = User.find_by(email: params[:email])
+    if user && !user.activated? && user.authenticated?(:activation, params[:id])
```

Comment:
You can use `&.` instead of `&&`.

Reply:
&. doesn't work when there are 3 conditions unless the second one also returns the user object. Also even though you could write user&.authenticated? && !user.activated?, since there are 3 conditions, I think it's more readable to have the consistency of using && between each.